### DISCUSSION TITLE
Accumulated output

### DISCUSSION THREAD

**Discussion by cbblanke** [NASA-LIS/LISF#1212] [Q&A] [answered]
In MODEL_OUTPUT.TBL you can specify if an output variable is instantaneous, time averaged, both, or accumulated.  I am doing Noah 3.3 runs over Africa and trying to output daily evapotranspiration (accumulated).  The units for ET ("Evap") are stated as (kg/m2s) but after accumulation (integration over time) it should be kg/m2 (equivalent to mm of water).  The max value in the file is about 0.004 which seems way too low--other sources give daily ET of 3-6 mm or so.   The NetCDF output file specifies the units in "kg m-2 s-1" which is not correct for an accumulation.  

I believe what it is doing is adding the current ET rate (kg/m2s) every model timestep (30 min) throughout the accumulation period (1 day).  So I multiplied the total by 1800 (the number of seconds in a timestep) to get the total kg/m2 ET in the day.  I ended up with a plot with a maximum ET of about 6, which is close to what I expected.

I suggest for accumulated quantities that the current rate (whatever per second) be multiplied by the number of seconds in the timestep before output.  In addition, the units on the output should not have the "per second" included.
Note that if I had interpreted the output as "kg/m2s" as the file says, it would still be incorrect because it is adding together 48 daily timesteps to get that value.

This was done with Noah 3.3 in LIS 7.1.

**Comment by dmocko** [comment, marked as answer]
Hi @cbblanke 

Thanks for the question.  I believe you have described things correctly.  This line in your MODEL_OUTPUT.TBL file probably looks similar to this:

`Evap:              1  kg/m2s   UP   3 0 0 1 139 10000   # Total Evapotranspiration (kg/m2s)`

The "3" in the line above tells LIS to write accumulated output.

You are correct that for the instance above, you need to multiply by the number of seconds in a timestep to get the total amount.

However, LIS is correctly doing exactly what you are telling it to do.  LIS is outputting the accumulation of the rate over your output interval.

If you want the daily accumulation, you can simply change the units to "kg/m2" and run again.

`Evap:              1  kg/m2   UP   3 0 0 1 139 10000   # Total Evapotranspiration (kg/m2s)`

Now LIS will output the accumulation of the amounts over your output interval, and the netCDF output file should list the units of "kg m-2" and have the magnitudes that you expect.

**Comment by cbblanke** [comment]
Thanks!  I did not realize you could switch the units to kg/m2.

**Reply by dmocko** [reply]
This may not be true for Noah-3.3 LSM in LIS-7.1.  I'm in a meeting today, and am unable to check.  But if it doesn't work for you, some simple code could be added that I can help you with.
